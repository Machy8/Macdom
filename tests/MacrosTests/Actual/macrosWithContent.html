js somescript.js
js-async https://code.jquery.com/jquery-3.2.1.js
charset utf-8
