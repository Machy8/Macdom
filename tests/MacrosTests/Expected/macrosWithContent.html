<script type="text/javascript" src="somescript.js"></script>
<script type="text/javascript" src="https://code.jquery.com/jquery-3.2.1.js" async="true"></script>
<meta charset="utf-8">
